workbench-core/datasets/src/dataSetService.test.ts

```diff
@@ -1590,25 +1616,28 @@ describe('DataSetService', () => {
 
   describe('listStorageLocations', () => {
     it('returns an array of known StorageLocations.', async () => {
-      await expect(dataSetService.listStorageLocations(mockAuthenticatedUser)).resolves.toMatchObject<
-        StorageLocation[]
-      >([
-        {
-          name: mockDataSetStorageName,
-          awsAccountId: mockAwsAccountId,
-          type: mockDataSetStorageType,
-          region: mockAwsBucketRegion
-        }
-      ]);
+      await expect(
+        dataSetService.listStorageLocations(mockAuthenticatedUser, 3, undefined)
+      ).resolves.toMatchObject<PaginatedResponse<StorageLocation>>({
+        data: [
+          {
+            name: mockDataSetStorageName,
+            awsAccountId: mockAwsAccountId,
+            type: mockDataSetStorageType,
+            region: mockAwsBucketRegion
+          }
+        ],
+        paginationToken: undefined
+      });
     });
     it('generates an audit event when an error is thrown', async () => {
       jest
         .spyOn(DdbDataSetMetadataPlugin.prototype, 'listStorageLocations')
         .mockRejectedValueOnce(new Error('intentional test error'));
 
-      await expect(dataSetService.listStorageLocations(mockAuthenticatedUser)).rejects.toThrowError(
-        new Error('intentional test error')
-      );
+      await expect(
+        dataSetService.listStorageLocations(mockAuthenticatedUser, 3, undefined)
```

Comment:
listStorageLocations will also need to be updated for pagination (but much lower priority until BYOB is enabled)

Reply:
I believe I updated that as well already in this PR?

Reply:
https://github.com/aws-solutions/solution-spark-on-aws/pull/884/files#diff-933c8bc4dd4957a4dfaba209a4a4fece8c46d317461b43e18ccb5820e79ab7c1R115